### PR TITLE
[DOCU-2049] Gateway Docker Install: add text inputs

### PR DESCRIPTION
**Pull request by falondarville**
### Summary

Add text inputs to address confusing around what the user names, versus what's a hard value. 

### Reason

Addresses [Issues 3384](https://github.com/Kong/docs.konghq.com/issues/3384)
Adds clarity to users

### Testing

Link TBA

### CODE REVIEW

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 1]
```diff
@@ -84,42 +80,27 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
...
-   ```
+   <pre><code>docker network create <div contenteditable="true">{NETWORK_NAME}</div></code></pre>
 
-2. Start a PostgreSQL container:
+1. Start a PostgreSQL container:
 
-   ```p
-   docker run -d --name kong-database \
-     --network=kong-net \
+   <pre><code>docker run -d --name <div contenteditable="true">{CONTAINER_NAME}</div> \
```
Comment (lena-larionova) [inline app/gateway/2.6.x/install-and-run/docker.md:87]:
```suggestion
   <pre><code>docker run -d --name <div contenteditable="true">{PG_CONTAINER_NAME}</div> \
```

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 2]
```diff
@@ -84,42 +80,27 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
...
- -e "KONG_PASSWORD=<div contenteditable="true">{PASSWORD}</div>" \
- kong-ee migrations bootstrap</code></pre>
-{% endnavtab %}
-{% navtab Kong Gateway (OSS) %}
-<pre><code>docker run --rm --network=kong-net \
+<pre><code>docker run --rm --network=<div contenteditable="true">{NETWORK_NAME}</div> \
  -e "KONG_DATABASE=postgres" \
- -e "KONG_PG_HOST=kong-database" \
- -e "KONG_PG_PASSWORD=kong" \
+ -e "KONG_PG_HOST=<div contenteditable="true">{CONTAINER_NAME}</div>" \
```
Comment (lena-larionova) [inline app/gateway/2.6.x/install-and-run/docker.md:100]:
```suggestion
 -e "KONG_PG_HOST=<div contenteditable="true">{PG_CONTAINER_NAME}</div>" \
```

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 3]
```diff
@@ -131,11 +112,11 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
 {% capture start_container %}
 {% navtabs codeblock %}
 {% navtab Kong Gateway %}
-<div class="copy-code-snippet"><pre><code>docker run -d --name kong-ee \
- --network=kong-net \
+<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{CONTAINER_NAME}</div> \
```
Comment (lena-larionova) [inline app/gateway/2.6.x/install-and-run/docker.md:115]:
```suggestion
<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{GATEWAY_CONTAINER_NAME}</div> \
```

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 4]
```diff
@@ -153,8 +134,8 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
  kong-ee</code></pre></div>
 {% endnavtab %}
 {% navtab Kong Gateway (OSS) %}
-<div class="copy-code-snippet"><pre><code>docker run -d --name kong \
- --network=kong-net \
+<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{CONTAINER_NAME}</div> \
```
Comment (lena-larionova) [inline app/gateway/2.6.x/install-and-run/docker.md:137]:
```suggestion
<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{GATEWAY_CONTAINER_NAME}</div> \
```

**Review by lena-larionova — APPROVED** [review]
LGTM, just added a couple of edits to differentiate the database container name from the Gateway container name.

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 5]
```diff
@@ -153,8 +134,8 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
...
+ --network=<div contenteditable="true">{NETWORK_NAME}</div> \
  -e "KONG_DATABASE=postgres" \
  -e "KONG_PG_HOST=kong-database" \
  -e "KONG_PG_USER=kong" \
```
Comment (lena-larionova) [inline app/gateway/2.6.x/install-and-run/docker.md:141]: should these ones also be editable, as they were in the snippets above?

File: app/gateway/2.6.x/install-and-run/docker.md [hunk 6]
```diff
@@ -153,8 +134,8 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
  kong-ee</code></pre></div>
 {% endnavtab %}
 {% navtab Kong Gateway (OSS) %}
-<div class="copy-code-snippet"><pre><code>docker run -d --name kong \
- --network=kong-net \
+<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{CONTAINER_NAME}</div> \
+ --network=<div contenteditable="true">{NETWORK_NAME}</div> \
  -e "KONG_DATABASE=postgres" \
  -e "KONG_PG_HOST=kong-database" \
```
Comment (falondarville) [inline app/gateway/2.6.x/install-and-run/docker.md:140]:
```suggestion
 -e "KONG_PG_HOST=<div contenteditable="true">{PG_HOST_NAME}</div>" \
```

File: app/gateway/2.6.x/install-and-run/docker.md (hunk 5 above)
Comment (falondarville) [inline app/gateway/2.6.x/install-and-run/docker.md:141]:
```suggestion
 -e "KONG_PG_USER=<div contenteditable="true">{PG_USER_NAME}</div>" \
```

Comment (falondarville) [reply app/gateway/2.6.x/install-and-run/docker.md]: added editable fields for these. Good catch 💯

**Pull request merged**